site/content/docs/user/configuration.md

```diff
@@ -161,6 +161,18 @@ nodes:
 - role: worker
 {{< /codeFromInline >}}
 
+You can also set a specific Kubernetes version by setting the `node`'s container image. You can find available image tags on the [releases page](https://github.com/kubernetes-sigs/kind/releases). Please use the `sha256` shasum for your desired kubernetes version, as seen in this example:
+
+{{< codeFromInline lang="yaml" >}}
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+- role: worker
+  image: kindest/node:v1.16.4@sha256:b91a2c2317a000f3a783489dfb755064177dbc3a0b2f4147d50f04825d016f55
```

Comment:
I wonder how hard it would be to put this in a data file as a follow up, so we can keep the example updated with a valid one for the current kind version.

can be a follow up